src/Xhgui/Controller/Run.php

```diff
@@ -9,6 +9,23 @@ public function __construct($app, $profiles, $watches)
         $this->_watches = $watches;
     }
 
+    public function homepage()
+    {
+        // TODO: how to fetch DIC from app?
+        global $di;
+
+        if (false !== ($site = $di['sites']->getFirstAvailable())) {
+            $url = $this->_app->urlFor('home', compact('site'));
+            // TODO: this does not work (?)
+            // $this->_app->redirect($url);
+
+            header('Location: '. $url);
+            exit;
```

Comment:
Problem here is that Twig is trying to render the current page here, from within `redirect()`. Probably trying to render the error page for the exception being thrown. Weird framework, this Slim. :smile: